src/store.ts

```diff
@@ -160,10 +160,10 @@ export function buildStore<
  * @param options
  */
 export function createStore<
-  Id extends string,
   S extends StateTree,
-  G extends Record<string, StoreGetter<S>>,
-  A extends Record<string, StoreAction>
+  Id extends string = string,
+  G extends Record<string, StoreGetter<S>> = Record<string, StoreGetter<S>>,
+  A extends Record<string, StoreAction> = Record<string, StoreAction>
```

Comment:
```suggestion
  G extends Record<string, StoreGetter<S>>,
  A extends Record<string, StoreAction>,
  Id extends string
```

Reply:
What I don't really like about the defaults is that you loose all typing if you only provide the State type in the generic `createStore<State>()`

I think it makes more sense to manually type the return of the function like said at https://github.com/posva/pinia/issues/85#issuecomment-631473019

Reply:
Yes, it's a second way to do. But personally, I prefer to use generic feature and not polluating the body of my function.